src/CommunityToolkit.Maui.Camera/Providers/CameraProvider.macios.cs

```diff
@@ -0,0 +1,93 @@
+﻿using AVFoundation;
+using CommunityToolkit.Maui.Core.Primitives;
+using CoreMedia;
+using CoreVideo;
+using UIKit;
+
+namespace CommunityToolkit.Maui.Core;
+
+partial class CameraProvider
+{
+	public partial ValueTask RefreshAvailableCameras(CancellationToken token)
+	{
+		AVCaptureDeviceType[] deviceTypes =
```

Comment:
this can be a static readonly collection, to avoid extra allocations

Reply:
Done ✅